### PR TITLE
Gloo e2e ec2 instannce

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: test/e2e/aws_ec2_test.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ sudo ./echoapp2 --port 80 &
 
 - Verify that you can reach the app
   - `curl` the app, you should see a help menu for the app
-```bash
+```bas
```
Comment (kdorosh) [inline test/e2e/aws_ec2_test.go:55]: let's put this back

File: test/e2e/aws_ec2_test.go [hunk 2]
```diff
@@ -170,7 +170,7 @@ var _ = Describe("AWS EC2 Plugin utils test", func() {
 		// TODO[test enhancement] - create an EC2 instance on demand (or auto-skip the test) if the expected instance is unavailable
 		// See notes in the header of this file for instructions on how to restore the instance
 		ec2Port := 80
-		ec2Url := fmt.Sprintf("http://%v:%v/metrics", strings.Join([]string{"52", "91", "199", "115"}, "."), ec2Port)
+		ec2Url := fmt.Sprintf("http://%v:%v/metrics", strings.Join([]string{"100", "24", "224", "6"}, "."), ec2Port)
```
Comment (kdorosh) [inline test/e2e/aws_ec2_test.go:173]: let's add a comment noting that this is an elastic IP, thus can be reused if the instance dies/is deleted

File: test/e2e/aws_ec2_test.go (hunk 1 above)
Comment (EItanya) [reply test/e2e/aws_ec2_test.go:55]: oops

**Pull request merged**